.changeset/nice-fireants-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@celo/celocli': patch
+---
+
+when account:new is called display the full bip44 derivation path in the output. before it would miss the last 2 positions.
```

Comment:
when account:new is called it displays the full bip44 derivation path in the output. Before it would miss the last 2 positions.